watertap/unit_models/ion_exchange_0D.py

```diff
@@ -411,11 +560,11 @@ def build(self):
             doc="Column diameter",
         )
 
-        self.col_vol_per = Var(
-            initialize=10,
-            bounds=(0.02, 70),
-            units=pyunits.m**3,
-            doc="Column volume",
+        self.col_height_to_diam_ratio = Var(
+            initialize=1,
+            bounds=(1, 100),
+            units=pyunits.dimensionless,
+            doc="Min ratio of bed depth to diameter",
```

Comment:
```suggestion
            doc="For stability, bounded bed depth to diameter ratio",
```

Reply:
This isn't _totally_ for stability, more to ensure you don't end up with pancake-shaped columns. But with a LB = 1, they can end up like cubes, which isn't ideal but I left it for stability.